flujo/validation/fixers.py

```
@@ -0,0 +1,430 @@
+}
+
+
+def _vt3_preview(file_path: str, report: Any) -> int:
+    try:
+        text = Path(file_path).read_text(encoding="utf-8")
+    except Exception:
+        return 0
+    import re as _re
+
+    count = 0
+    for bad, good in _VT3_MAP.items():
```

Ruff (0.12.2)

Loop control variable `good` not used within loop body

Rename unused `good` to `_good`

(B007)